zen-news-summarization/README.md

```diff
@@ -0,0 +1,327 @@
+namely `collect`, `summarize` and `report`. The first step is responsible 
+for collecting articles, the second step summarizes them, and the last step 
+creates a report and posts it.
+- **The steps**: There is a concrete implementation for each step defined above.
+  - For the `collect` step, we have the [`bbc_news_source`](src/zennews/steps/sources/bbc.py)
+  which (on default) collects the top stories from the BBC news feed and 
+  prepares [`Article`](src/zennews/models/article.py) objects. 
+  - For the `summarize` step, we have implemented [`bart_large_cnn_samsum`](src/zennews/steps/summarize/bart_large_cnn_samsum.py)
+  step. As the name suggests, it uses bart model to generate summaries. 
+  - Ultimately, for the `report` step, we have implemented the 
+  [`post_summaries`](src/zennews/steps/report/report.py) 
+  step. It showcases how a generalized step can function within a ZenML 
+  pipeline and uses an alerter to share the results.
+- **The materializer**: As mentioned above, the steps within our pipeline are 
+using the concept of `Article`s to define their input and output space. Using 
+the [`ArticleMaterializer`](src/zennews/materializers/article_materializer.py), 
+we can show how to handle the materialization of these artifacts 
+when it comes to a data type that is not built-in.
+- **The custom stack component**: The ultimate goal of `ZenNews` is to 
+serve the use the direct outcomes of the pipeline. That is why we have used it
+as a chance to show the extensibility of ZenML in terms of the stack components 
+and implemented a [`DiscordAlerter`](src/zennews/alerter/discord_alerter.py).
+- **The CLI application**: The example also includes a 
+[Click](https://click.palletsprojects.com/en/8.1.x/) CLI application. 
+It utilizes how easily you can use our Python SDK to build your application 
+around your ZenML workflows. In order to see it in action simply execute:
+
+  ```bash
+  zennews --help 
+   ```
+
+# 🕹 Test it locally right away
+
+Once you installed the `zennews` package, you are ready to test it out locally 
+right away. The following command will get the top five articles from the BBC
+news feed, summarize them and present the results to you. 
+
+> Warning: This will temporarily switch your active ZenML stack to the 
+> **default** stack and when the pipeline runs, you will download the model 
+> to your local machine.
+
+```bash
+zennews bbc
+```
+
+You can also parameterize this process. In order to see the possible 
+parameters, please use:
+
+```bash
+zennews bbc --help
+```
+
+# 🚀 Switching to scheduled pipelines with Vertex
+
+The potential of an application like `zennews` can be only unlocked by 
+scheduling summarization pipelines instead of manually triggering them 
+yourself. In order to showcase it, we will set up a fully remote GCP stack 
+and use the `VertexOrchestrator` to schedule the pipeline.
+
+## Deploy ZenML on GCP
+
+Before you start building the stack, you need to deploy ZenML on GCP. For more 
```

Comment:
Do we need to say 'deploy ZenML server' here instead?

Reply:
I wanted to stay consistent with the docs. That's why I decided to avoid the `server` here.